### PR TITLE
If the model is not known, let the cost be None.

### PR DESCRIPTION
**Pull request by madiator**
Before:
```
│ Costs                      │                                                     │
│ Total Cost                 │ $0.000                                              │
│ Average Cost per Request   │ $0.000                                              │
│ Input Cost per 1M Tokens   │ $0.000                                              │
│ Output Cost per 1M Tokens  │ $0.000                                              │
```

After:
```
│ Costs                      │                                                     │
│ Total Cost                 │ $0.000                                              │
│ Average Cost per Request   │ $0.000                                              │
│ Input Cost per 1M Tokens   │ N/A                                                 │
│ Output Cost per 1M Tokens  │ N/A                                                 │
```

We should also not display cost as $0.0, but as N/A.

### CODE REVIEW

File: tests/unittests/test_online_status_tracker.py [hunk 1]
```diff
@@ -78,3 +79,41 @@ def test_online_status_tracker_final_stats():
...
+        tracker = OnlineStatusTracker()
+        tracker.model = "test-model"
+        # Should not raise KeyError and should set costs to None
+        assert tracker.input_cost_per_million is None
+        assert tracker.output_cost_per_million is None
+        assert "N/A" in tracker.input_cost_str
+        assert "N/A" in tracker.output_cost_str
+
+
+def test_online_status_tracker_external_model_cost_error():
```
[the remark this answers is not in the document]
Comment (madiator) [reply tests/unittests/test_online_status_tracker.py:96]: They are testing for different things. One is testing if the function throws a KeyError and another tests when the function returns an empty dictionary..

[the remark this answers is not in the document]
Comment (kartik4949) [reply tests/unittests/test_online_status_tracker.py:96]: @madiator 
Actually both the tests are not valid  for current `external_model_cost`

```
def external_model_cost(model, completion_window="*", provider="default"):
    """Get the cost of the model from the external providers registered."""
    if provider not in _DEFAULT_COST_MAP["external"]["providers"]:
        return {"input_cost_per_token": None, "output_cost_per_token": None}
    provider_cost = _DEFAULT_COST_MAP["external"]["providers"][provider]["cost"]
    if model not in provider_cost:
        raise ValueError(f"Model {model} is not supported by {provider}.")

    if completion_window not in provider_cost[model]["input_cost_per_million"]:
        raise ValueError(f"Completion window {completion_window} is not supported for {model} by {provider}.")

    cost = provider_cost[model]["input_cost_per_million"][completion_window]
    return {"input_cost_per_token": cost / 1e6, "output_cost_per_token": cost / 1e6}
    
 ```
 
 
 Above is `external_model_cost` definition and by any combination of model and provider argument we cannot have a situation of KeyError or {} return
 
 
 Either change `external_model_cost` with KeyError exception or we need to update the tests
 
 I can do it if you want 
 Thanks

[the remark this answers is not in the document]
Comment (madiator) [reply tests/unittests/test_online_status_tracker.py:96]: thanks for fixing the test
